packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/index.md

```diff
@@ -27,26 +27,43 @@ Add authentication using the Okta [redirect model](https://developer.okta.com/do
 
 ## Set up Okta
 
-Set up your [Okta org](/docs/concepts/okta-organizations/). The CLI is by far the quickest way to work with your Okta org, so we recommend using it for the first few steps. If you don't want to install the CLI, you can [manually sign up for an org](https://developer.okta.com/signup/) instead. We provide non-CLI instructions along with the CLI steps below.
+Set up your [Okta org](/docs/concepts/okta-organizations/). The Okta command-line interface (CLI) is the quickest way to work with your Okta org, so we recommend using it for the first few steps. If you don't want to install the CLI, you can [manually sign up for an org](https://developer.okta.com/signup/) instead.
 
-1. Install the Okta command-line interface: [Okta CLI](https://cli.okta.com/).
-2. If you don't already have a free Okta developer account, create one by entering `okta register` on the command line.
-3. Make a note of the Okta Domain as you need that later.
-4. **IMPORTANT:** Set the password for your Okta developer org by opening the link that's shown after your domain is registered. Look for output similar to this:
+1. Install [Okta CLI](https://cli.okta.com/).
+1. If you don't already have a free Okta developer account:
```

Comment:
We have this caveat here, but removed it in the first file (protect-your-api/main/index.md) - should we include this in the other topic as well?

Reply:
I don't think it's strictly necessary because the messaging is slightly different, but I've added it anyway.